participantes/NathanFirmo/compose.yaml

```
@@ -0,0 +1,62 @@
+name: rinha-de-backend
+
+services:
+  api01: &api
+    image: nathanfirmo/rinha-de-backend-2024-q1:latest
+    hostname: api01
+    environment:
+      - DATABASE_URL=postgres://admin:admin@db:5432/rinha
+    depends_on:
+      db:
+        condition: service_healthy
+    deploy:
+      resources:
+        limits:
+          cpus: '0.2'
+          memory: '25MB'
+    restart: always
+
+  api02:
+    <<: *api
+    hostname: api02
+
+  nginx:
+    image: nginx:1.25.3-alpine3.18
+    volumes:
+      - ./config/nginx.conf:/etc/nginx/nginx.conf:ro
+    depends_on:
+      - api01
+      - api02
+    ports:
+      - '9999:9999'
+    deploy:
+      resources:
+        limits:
+          cpus: '0.2'
+          memory: '100MB'
+
+  db:
+    image: postgres:16.2-bullseye
+    hostname: db
```

Mellhor fixar os nomes dos containers.
```suggestion
    hostname: db
    container_name: db
```